cursive-core/src/utils/markup/markdown.rs

```diff
@@ -154,58 +265,127 @@ mod tests {
-                },
-                Span {
-                    content: " love ",
-                    width: 6,
-                    attr: &Style::none(),
-                },
-                Span {
-                    content: "Cursive",
-                    width: 7,
-                    attr: &Style::from(Effect::Bold),
-                },
-                Span {
-                    content: "!",
-                    width: 1,
-                    attr: &Style::none(),
-                }
-            ]
-        );
+        let style_none = Style::none();
+        let style_bold = Style::from(Effect::Bold);
+        let style_italic = Style::from(Effect::Italic);
+
+        let verify_spans = vec![
+            Span {
+                content: "# ",
+                width: 2,
+                attr: &style_bold,
+            },
+            Span {
+                content: "Attention",
+                width: 9,
+                attr: &style_bold,
+            },
+            Span {
+                content: "\n\n",
+                width: "\n\n".width(),
+                attr: &style_none,
+            },
+            Span {
+                content: "I ",
+                width: 2,
+                attr: &style_none,
+            },
+            Span {
+                content: "really",
+                width: 6,
+                attr: &style_italic,
+            },
+            Span {
+                content: " love ",
+                width: 6,
+                attr: &style_none,
+            },
+            Span {
+                content: "Cursive",
+                width: 7,
+                attr: &style_bold,
+            },
+            Span {
+                content: "!",
+                width: 1,
+                attr: &style_none,
+            },
+            Span {
+                content: "\n\n> ",
+                width: "\n\n> ".width(),
+                attr: &style_none,
+            },
+            Span {
+                content: "This is a blockquote",
+                width: 20,
+                attr: &style_none,
+            },
+            Span {
+                content: "\n\n",
+                width: "\n\n".width(),
+                attr: &style_none,
+            },
+            Span {
+                content: "fn main() {\n    println!(\"Goodbye, world!\");\n}\n",
+                width: "fn main() {\n    println!(\"Goodbye, world!\");\n}\n".width(),
+                attr: &style_none,
+            },
+            Span {
+                content: "\n\n",
+                width: "\n\n".width(),
+                attr: &style_none,
+            },
+            Span {
+                content: "<https://en.wikipedia.org> ",
+                width: "<https://en.wikipedia.org> ".width(),
+                attr: &style_none,
+            },
+            Span {
+                content: "Link",
+                width: "Link".width(),
+                attr: &style_none,
+            },
```

Comment:
Does that mean `[link](url)` is rendered `<url> link`? Is that the intended behavior?